src/dsp-services.js

```diff
@@ -0,0 +1,170 @@
+import {json} from "./store.js";
+import {language} from './store';
+import {get} from 'svelte/store'
+const j = get(json);
+const ontologyIri = j['DSP']['Ontology'];
+const server = j['DSP']['Server'];
+const shortCode = j['DSP']['ShortCode'];
+let ontology;
+export class ListNode {
+    constructor(label, id) {
+        this.label = label;
+        this.id = id;
+    }
+}
+/*
+TODO: Need to implement a general getter for Prop and Res, where one can pass what to return via argument
+ */
+export async function getOntology(){
+    const res = await fetch('https://' + server + '/v2/ontologies/allentities/' + encodeURIComponent('http://' + server + '/ontology/' + shortCode + '/' + ontologyIri + '/v2') + '?allLanguages=true', {
```

Comment:
If you like you can use template strings -> [](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals)

Reply:
I'll leave it as it is for now in case these methods need to be exported to another tool/language but might refactor it in the future